ajax/libs/bodymovin/package.json

```diff
@@ -15,13 +15,13 @@
     "export"
   ],
   "license": "MIT",
-  "homepage": "https://github.com/bodymovin/bodymovin/wiki",
+  "homepage": "https://github.com/airbnb/lottie-web/blob/master/LICENSE.md",
```

Comment:
The homepage should be https://airbnb.io/lottie/

Reply:
Updated.